### PR TITLE
Add 0.7.0 release notes

### PR DESCRIPTION
**Pull request by yaron2**

### CODE REVIEW

File: docs/release_notes/v0.7.0.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+
+#### Post installation
+
+After Dapr 0.7.0 has been installed, perform a rolling restart for your deployments.
+
+## Breaking Changes
+
+### gRPC Service Invocation
+
+### Tracing configuration
```
Comment (shalabhms) [inline docs/release_notes/v0.7.0.md:237]: With 0.7.0 release, Dapr trace configuration is simplified. User now has to provide "samplingRate" to enable or disable the traces.  Set "samplingRate: "0" to disable the traces. Set "samplingRate: "1" to  always generate the the traces. Refer https://github.com/dapr/docs/blob/master/concepts/observability/traces.md for more details.

File: docs/release_notes/v0.7.0.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
+  
+# Dapr 0.7.0
+ 
+We're happy to announce the release of Dapr 0.7.0! 
+
+This release adds new components, and focuses on improvements to tracing and service invocation.
+We also added a performance test for service invocation, fixed bugs, improved stability and added features to our long haul tests infrastructure.
```
Comment (mchmarny) [inline docs/release_notes/v0.7.0.md:7]: With the v0.7 release we focused on improvements to tracing and service invocation. This release also expands Dapr functionality with addition components. Continuing with the team of improved stability, we also added a performance test for service invocation, fixed bugs, and added features to our long haul tests infrastructure.

While we focused in this release on improvements to tracing and service invocation, we did also extend Dapr functionality with addition of new components , and 
We also added a performance test for service invocation, fixed bugs, improved stability and added features to our long haul tests infrastructure.

Adds extends Dapr functionality or introduces ... using

Comment (yaron2) [reply docs/release_notes/v0.7.0.md:7]: What does "Continuing with the team of improved stability?"

Comment (mchmarny) [reply docs/release_notes/v0.7.0.md:7]: The following sentence in the original had the bit about `improved stability`. As Dapr is nearing v1.0 release I think we should start communicating A) the growing focus on stability B) impact of the customer validation. I'm coming late to this so feel free to ignore it

File: docs/release_notes/v0.7.0.md [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+dapr init --kubernetes
+```
+
+#### Post installation
+
+After Dapr 0.7.0 has been installed, perform a rolling restart for your deployments.
+
+## Breaking Changes
+
+### gRPC Service Invocation
```
Comment (youngbupark) [inline docs/release_notes/v0.7.0.md:234]: ### gRPC Protobuf update

* Package name in Dapr gRPC protobuf files has been changed to :
   - renamed to `dapr to `dapr.proto.dapr.v1`
   - renamed to `daprclient` - `dapr.proto.daprclient.v1` 
   - moved the common protobuf message related to service invocation to `dapr.proto.common.v1` package

### gRPC Service invocation

* If you're a Java SDK or dotnet SDK user, you must upgrade your SDK to the latest version. 
* If you're a JS/Python/CPP/Rust SDK user, you must upgrade your SDK to the latest version first and update your code by referring to each sdk's examples.
  - [javascript example](https://github.com/dapr/js-sdk/blob/master/examples/simple/app.js)
  - [python example](https://github.com/dapr/python-sdk/tree/master/examples/invoke-simple)
  - [cpp-sdk example](https://github.com/dapr/cpp-sdk/tree/master/examples/echo_app)
  - [rust-sdk example](https://github.com/dapr/rust-sdk/tree/master/examples)
* If HTTP callee app returns non-OK status code to caller app using gRPC API, caller app's gRPC client will throw the exception in 0.7.0 release. The caller app needs to handle the exception properly.

**Pull request merged**